### PR TITLE
Adding Homebrew package

### PR DESCRIPTION
**Pull request by theabrar**
This is for https://github.com/hashicorp/terraform-cdk/issues/65. Raised a PR on homebrew https://github.com/Homebrew/homebrew-core/pull/60113 for the same.
Also added a release_homebrew workflow, which should kick-off if the commit message has 'Bump version' in it, needs a committer token to automatically create a PR on homebrew after a release.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -72,4 +72,27 @@ jobs:
...
+          version=$(node -p "require('./package.json').version")
+          echo "::set-output name=version::${version}"
+      # A PR will be sent to github.com/Homebrew/homebrew-core to update this formula:
+      - uses: mislav/bump-homebrew-formula-action@v1
+        with:
+          formula-name: cdktf
+          download-url: https://registry.npmjs.org/cdktf-cli/-/cdktf-cli-${{ steps.get_version.outputs.version }}.tgz
+          commit-message: cdktf ${{ steps.get_version.outputs.version }}
+        env:
+          COMMITTER_TOKEN: ${{ secrets.HOMEBREW_COMMITTER_TOKEN }}
```
Comment (anubhavmishra) [inline .github/workflows/release.yml:98]: I will generate a GitHub token with "public" scope for this one.

**Comment by anubhavmishra** [comment]
Waiting on the homebrew PR to be merged before we approve this one. Homebrew/homebrew-core#60113

**Comment by theabrar** [comment]
It’s merged!

File: .github/workflows/release.yml (hunk 1 above)
Comment (skorfmann) [reply .github/workflows/release.yml:98]: Has this token been set already?

File: docs/getting-started/python.md [hunk 2]
```diff
@@ -9,6 +9,13 @@
 
 ### Install CDK for Terraform CLI
 
+Install with [Homebrew](https://brew.sh):
```
Comment (jsteinich) [inline docs/getting-started/python.md:12]: Might be worth pointing out that this method is mac (I suppose or Linux) specific.

File: .github/workflows/release.yml (hunk 1 above)
Comment (anubhavmishra) [reply .github/workflows/release.yml:98]: I have saved the token for hashicorp-ci GitHub user.

**Pull request merged**